### ISSUE TITLE
Predict on incomplete glyph set

### ISSUE THREAD

**Issue by matheusrhoden** [yizhiwang96/deepvecfont#32]
Hi! 

Is it possible to run the model against an incomplete glyph set? 

For example, drawing a few glyphs myself and then having DVF generate the rest of the alphabet.

Thanks!

**Comment by yizhiwang96** [comment]
Yes, it is exactly the '**few-shot generation**' feature demonstrated in our paper. The reference number was 4 in our paper, and theoretically it can be any number, e.g., the number of your drawn glyphs.

**Comment by matheusrhoden** [comment]
@yizhiwang96 Got it, thanks. Just to clarify, though, do I have to (re)train the model in order to use different reference glyphs? Apparently the modality fusion checkpoint shape has to match the number of reference glyphs
